src/Skoruba.IdentityServer4.STS.Identity/Helpers/UserResolver.cs

```diff
@@ -15,17 +16,36 @@ public UserResolver(UserManager<TUser> userManager, LoginConfiguration configura
             _policy = configuration.ResolutionPolicy;
         }
 
-        public async Task<TUser> GetUserAsync(string login)
+        public async Task<TUser> GetUserAsync(string login, string tenant = null)
         {
-            switch (_policy)
+            if (_userManager.IsMultiTenant())
             {
-                case LoginResolutionPolicy.Username:
-                    return await _userManager.FindByNameAsync(login);
-                case LoginResolutionPolicy.Email:
-                    return await _userManager.FindByEmailAsync(login);
-                default:
-                    return null;
+                switch (_policy)
+                {
+                    case LoginResolutionPolicy.Username:
+                        return await _userManager.FindByNameAsync(login, tenant);
```

Comment:
probubly bug here, signature is
`
public Task<TUser> FindByEmailAsync(string tenantCode, string email)
 `
so should be
`
return await _userManager.FindByNameAsync(tenant, login);
`